packages/path_provider/path_provider_android/pigeons/messages.dart

```diff
@@ -41,4 +41,6 @@ abstract class PathProviderApi {
   List<String?> getExternalCachePaths();
   @TaskQueue(type: TaskQueueType.serialBackgroundThread)
   List<String?> getExternalStoragePaths(StorageDirectory directory);
+  @TaskQueue(type: TaskQueueType.serialBackgroundThread)
+  String? getDownloadsPath();
```

Comment:
I think this change is causing the test failures since `getDownloadsPath()` doesn't need to be implemented on the native side.